pkg/ctree/tree_test.go

```diff
@@ -219,6 +219,21 @@ func TestConfigTree(t *testing.T) {
 			}, ShouldPanic)
 		})
 
+		Convey("doesn't panic on ns where the root and ns don't have a policy", func() {
+			d1 := newMockNode()
+			d1.data = "a"
+			d2 := newMockNode()
+			d2.data = "b"
+			c := New()
+			c.Add([]string{"intel", "foo", "sdi-x", "cody"}, d1)
+			c.Add([]string{"intel", "foo", "sdi-x", "nan"}, d2)
+			c.Freeze()
```

Comment:
I believe Get calls freeze for you on the first get so I'm not sure this is needed.